src/main.tf

```diff
@@ -44,18 +50,12 @@ resource "azurerm_kubernetes_cluster" "main" {
   }
 
   default_node_pool {
-    #bridgecrew:skip=BC_AZR_GENERAL_97:Skipping `Ensure that AKS use the Paid Sku for its SLA`
```

Comment:
FYI, when we integrate checkov scanning, we will likely use a similar syntax to filter policies: https://www.checkov.io/2.Basics/Suppressing%20and%20Skipping%20Policies.html

Reply:
That's fine. It seems like these are specific to Bridgecrew. We can add the Checkov ones back when needed.